.circleci/config.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       # run tests
       - run: npm test -- --maxWorkers=4
       # send to coveralls
-      - run: cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+      - run: cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Was this giving you issues? I think we should keep this last line

Reply:
added back